pkl-core/src/main/java/org/pkl/core/runtime/VmUtils.java

```diff
@@ -101,7 +102,7 @@ public static MaterializedFrame createEmptyMaterializedFrame() {
   }
 
   public static Context createContext(Runnable initializer) {
-    var context = Context.newBuilder("pkl").engine(PKL_ENGINE).build();
+    var context = PKL_ENGINE.get().build();
```

Comment:
`VmUtils` now consults the `VmEngineManager` to obtain the engine; in doing so, the `onContextCreated` plugin hook event is dispatched.